modules/profilereview/src/Auth/Process/ProfileReview.php

```diff
@@ -110,7 +110,7 @@ public static function validateConfigValue($attribute, $value, $logger)
      * @param array $state The state data.
      * @return mixed The attribute value, or null if not found.
      */
-    protected function getAttribute($attributeName, $state)
+    protected function getAttribute(string $attributeName, array $state): mixed
```

Comment:
Have you checked if this is an overriding function and that the typing matches the underlying library's definition of getAttribute? (generalize for all functions you are strong typing)

Reply:
No, it is not. When I did the upgrade to SimpleSAMLphp version 2, I had to strong type everything that was inherited because it was done in the library. So the changes I'm making now are not on inherited methods. Great question.

Reply:
Then you should NOT do that.

Reply:
Only strong type if the inherited functions are strong typed.